pkgs/development/python-modules/pyqt/5.x.nix

```
@@ -116,7 +132,20 @@ in buildPythonPackage rec {
     EOF
   '';
 
-  installCheckPhase = let
+  ${if isPy3k then "pythonImportsCheck" else null} = [
+    "PyQt5"
+    "PyQt5.QtCore"
+    "PyQt5.QtQml"
+    "PyQt5.QtWidgets"
+    "PyQt5.QtGui"
+  ]
```

more canonical way of expressing this
```suggestion
  pythonImportsCheck = lib.optionals isPy3k ([
    "PyQt5"
    "PyQt5.QtCore"
    "PyQt5.QtQml"
    "PyQt5.QtWidgets"
    "PyQt5.QtGui"
  ]
  ....
  );
```

---

Changed. I have further simplified this part, `pythonImportsCheck` seems to work with Python 2.